### PR TITLE
riseupvpn: reduce false positives

### PR DESCRIPTION
**Pull request by cyBerta**
part of  https://github.com/ooni/probe/issues/1354

### CODE REVIEW

**Comment by cyBerta** [comment]
tests are still missing

**Comment by bassosimone** [comment]
🆗 👀 please ping us when it's ready! thank you for your work so far! :-)

**Comment by cyBerta** [comment]
@bassosimone I think, I'm mostly done. I still need to resolve a merge conflict and refactor the code a little bit more, but these won't be fundamental changes.

**Comment by cyBerta** [comment]
I might also rearrange the commit history, because I took some unnecessary loops during the development.

**Comment by bassosimone** [comment]
@cyBerta no pressure, I thought you were done because I saw you requested a review! Ping me when ready! :-)

**Comment by cyBerta** [comment]
@bassosimone ok, now it's ready for review ;)

File: internal/engine/experiment/riseupvpn/riseupvpn.go [hunk 1]
```diff
@@ -96,6 +97,7 @@ func (tk *TestKeys) UpdateProviderAPITestKeys(v urlgetter.MultiOutput) {
 }
 
 // AddGatewayConnectTestKeys updates the TestKeys using the given MultiOutput result of gateway connectivity testing.
+// sets TransportStatus to "ok" if any successful TCP connection could be made
```
Comment (bassosimone) [inline internal/engine/experiment/riseupvpn/riseupvpn.go:100]:
```suggestion
// Sets TransportStatus to "ok" if any successful TCP connection could be made
```

**Review by bassosimone — APPROVED** [review]
Thank you!

:whale:

**Comment by bassosimone** [comment]
Great work!

**Pull request merged**